src/Softeq.NetKit.Chat.SignalR/Sockets/ChannelSocketService.cs

```diff
@@ -91,13 +106,24 @@ public async Task<ChannelSummaryResponse> CreateDirectChannelAsync(CreateDirectC
 
         public async Task<ChannelSummaryResponse> UpdateChannelAsync(UpdateChannelRequest request)
         {
+            var updatingChannel = await _channelService.GetChannelSummaryAsync(request.SaasUserId, request.ChannelId);
+
             await _channelService.UpdateChannelAsync(request);
 
-            var channelSummary = await _channelService.GetChannelSummaryAsync(request.SaasUserId, request.ChannelId);
+            var updatedChannel = await _channelService.GetChannelSummaryAsync(request.SaasUserId, request.ChannelId);
```

Comment:
do we need this additional call? This is dapper. It's so dummy that it will make real call to db